installer/conf/kube.conf

```diff
@@ -119,8 +125,8 @@
       max_retry_wait 9m
      </match>
 
-     <match oms.api.ContainerNodeInventory**>
-      type out_oms_api
+     <match oms.containerinsights.ContainerNodeInventory**>
+      type out_oms
```

Comment:
why are you changing this ?Rashmi fixed this couple weeks back. Can you first merge from ci_feature into your branch and then apply your changes & create pr ?